datanode/sqlstore/migrations/0001_initial.sql

```diff
@@ -9,6 +9,34 @@ create table blocks
     height        BIGINT                   NOT NULL,
     hash          BYTEA                    NOT NULL
 );
+select create_hypertable('blocks', 'vega_time', chunk_time_interval => INTERVAL '1 day');
+create index on blocks (height);
+
+create table last_block
+(
+    vega_time     TIMESTAMP WITH TIME ZONE NOT NULL PRIMARY KEY,
+    height        BIGINT                   NOT NULL,
+    hash          BYTEA                    NOT NULL
+);
+
+-- +goose StatementBegin
+CREATE OR REPLACE FUNCTION update_last_block()
+    RETURNS TRIGGER
+    LANGUAGE PLPGSQL AS
+$$
+BEGIN
+    Delete from last_block;
```

Comment:
maybe add a one_row_check like chain then you can just update instead of insert/delete